setup.py

```diff
@@ -56,5 +53,10 @@
         'Topic :: Terminals :: Serial',
     ],
     platforms = 'any',
-    scripts = ['serial/tools/miniterm.py'],
+    entry_points={
+        'console_scripts': [
+            'list_serial_ports = serial.tools.list_ports:main',
```

Comment:
Several reasons:
1. To use `python -m serial.tools.list_ports` one must read the docs. list_serial_ports is discovered by using bash completion and be inspecting the package. I added this one after looking up the former for the nth time in the docs.
2. list_serial ports can be used even if a virtualenv is active.